.pipelines/shared-cluster.yml

```diff
@@ -0,0 +1,97 @@
+# No PR triggers as it runs on a schedule
+pr: none
+trigger: none
+
+schedules:
+- cron: "0 1 * * Sun"
+  displayName: ⭕ Rebuild shared cluster Sun 1am (UTC)
+  always: true
+  branches:
+    include:
+    - master
+
+variables:
+  LOCATION: eastus
+  CLUSTER: aro-v4-shared
+
+jobs:
+- job: ⏬ Tear down shared cluster
+  pool:
+    vmImage: "Ubuntu-16.04"
+
+  container:
+    image: mcr.microsoft.com/azure-cli:latest
```

Comment:
this looks like it might be better than what we're doing elsewhere currently (grep template-setup-azure-tools.yml).  For consistency can you match what we do elsewhere for now, but I would be interested in an issue + follow-up PR if there's an opportunity to simplify.

Reply:
I feel like these are 2 different things: one way (template-setup-azure-tools.yml) does test the python extension in the same repo, while this one (using the docker image) mocks what the end user will end up doing, I could've installed via pip/etc but this one seemed faster.